libvast/CMakeLists.txt

```diff
@@ -236,6 +242,56 @@ install(
 
 add_library(vast::libvast ALIAS libvast)
 
+# ----------------------------------------------------------------------------
+# flattbuffers
+# ----------------------------------------------------------------------------
+
+file(GLOB flatbuffers_schemas ${CMAKE_CURRENT_SOURCE_DIR}/vast/fbs/*.fbs)
+set(flatbuffers_output_path ${CMAKE_CURRENT_BINARY_DIR}/vast/fbs)
+
+add_custom_target(flatbuffers)
+add_dependencies(libvast flatbuffers)
+
+# Translate paths to desired output paths.
+foreach (schema ${flatbuffers_schemas})
+  get_filename_component(basename ${schema} NAME_WE)
```

Comment:
We should extract the body of this loop into a function and move it to `cmake/Flatbuffers.cmake`.

Reply:
I don't think this would help here, because we are adding targets in here dynamically, one for each file.

Reply:
Per our discussion, you convinced me that the extra safety belt makes sense here.